File: Highest Scoring Word/Highest Scoring Word.py
import string
def high(x):
    result = {}
    for s1 in x.split(" "):
        tmp = []
        for s2 in s1:
            tmp.append(string.ascii_lowercase.index(s2)+1)
        result[s1]=sum(tmp)
    return sorted(result.items(),key=lambda x:-x[1])[0][0]
            
    

File: Highest Scoring Word/Highest Scoring Word.ts
export const high = (text: string): string => {
  const result = text.split(" ").map(word => word.split("").map(s => s.charCodeAt(0)-96).reduce((a,b) => a+b,0))
  const result_hi = result.indexOf(Math.max.apply(null,result))
  return text.split(" ")[result_hi]
}
